manifests/0000_70_dns-operator_02-deployment.yaml

```diff
@@ -36,6 +36,31 @@ spec:
         resources:
           requests:
             cpu: 10m
+      - name: kube-rbac-proxy
+        image: quay.io/openshift/origin-kube-rbac-proxy:latest
+        args:
+        - --logtostderr
+        - -v=8
```

Comment:
Will this log stuff on every scrape interval? If so is that ok?

Reply:
It will.  I'll delete it.